src/padplus-manager/api-request/contact.ts

```diff
@@ -11,6 +14,124 @@ export class PadplusContact {
   constructor (requestClient: RequestClient) {
     this.requestClient = requestClient
   }
+
+  public async newTag (tag: string): Promise<string> {
+    log.verbose(PRE, `newTag(${tag})`)
+
+    const data = {
+      tag,
+    }
+    const result = await this.requestClient.request({
+      apiType: ApiType.CREATE_LABEL,
+      data,
+    })
+    if (result) {
+      const labelStr = result.getData()
+      if (labelStr) {
+        const label = JSON.parse(labelStr)
```

Comment:
We need to create a type for the `label` here, which is the grpc server response data type.